API/models-json/Open-Ethics-Data-Passport.json

```diff
@@ -0,0 +1,46 @@
+{
+  "title": "Open Ethics Data Passport",
+  "type": "object",
+  "description": "OEDP Model",
```

Comment:
@OpenEthicsAI @abhi12ravi I'm noticing that my previous definitions are misleading as i referred to the data model in the structural sense and I don't want to confuse with the AI models which we want to describe with the schemas, so we'd change the naming to the data schema